src/components/new-request/search.vue

```diff
@@ -32,9 +39,12 @@
               </div>
             </v-list-item-content>
 
+            <!-- render but conditionally hide disabled list items, so that the v-select
+            continues to display the current selection even when a different group is active -->
```

Comment:
If the item is not in the list then the v-select sometimes shows a blank selection (ticket 17044).